### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by izdeby**
Updated readme for the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+# Prerequisites
+You will need this packages to be installed:
+- ipython
+- notebook
+- pytorch
+- torchvision
+
+If you have conda installed on your machine, you can install these via
+```
+conda install ipython pytorch notebook torchvision cpuonly -c pytorch-nightly
```
Comment (cpuhrsch) [inline README.md]: Users might want to use GPU. You could link to the main page.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
-
+Please note that current project is in active development and everything here is a subject to change.
 Currently migrating code from other sources.
-
 ![alt text](assets/sign.png "Construction sign")
+
+
+
+# Prerequisites
+You will need this packages to be installed:
```
Comment (cpuhrsch) [inline README.md]: Outside of pytorch neither of these are strictly required. torchvision is never imported and ipython + notebook is only needed for the examples.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+- notebook
+- pytorch
+- torchvision
+
+If you have conda installed on your machine, you can install these via
+```
+conda install ipython pytorch notebook torchvision cpuonly -c pytorch-nightly
+```
+
+# Build
```
Comment (cpuhrsch) [inline README.md]: You could add some sentences about ```pip install -e .``` as well. Also, this is for the debug build.

**Comment by cpuhrsch** [comment]
If we're adding this info we might as well remove the construction sign and add a bunch more general overview information.

File: README.md (hunk 2 above)
Comment (izdeby) [reply README.md]: It wont compile without torchvision. 
Usually prerequisites list everything that you need to have the whole project working and not just some parts?

**Comment by cpuhrsch** [comment]
Resolved by https://github.com/pytorch/nestedtensor/pull/10

**Pull request closed without merging**